omnibus/resources/chefdk/msi/localization-en-us.wxl.erb

```diff
@@ -3,7 +3,7 @@
   <!-- http://wix.codeplex.com/SourceControl/changeset/view/792e101c5cf7#src%2fext%2fUIExtension%2fwixlib%2fWixUI_en-us.wxl -->
   <String Id="LANG">1033</String>
   <String Id="ProductName"><%= friendly_name %></String>
-  <String Id="ManufacturerName"><%= maintainer.encode(:xml => :attr) %></String>
```

Comment:
I think `:attr` should be `:text` since it is being injected as element text and not an attribute value